sequence_processing_pipeline/Pipeline.py

```diff
@@ -459,21 +459,50 @@ def get_sample_names(self, project_name=None):
 
     def _get_sample_names_from_sample_sheet(self, project_name):
         if project_name is None:
-            return [x.Sample_Name for x in self.sample_sheet.samples]
+            if 'orig_name' in self.sample_sheet.samples[0]:
+                # return set as a list to match non-replicate results.
+                return sorted(list({x.orig_name for x in
+                                    self.sample_sheet.samples}))
```

Comment:
```suggestion
                return sorted({x.orig_name for x in
                                    self.sample_sheet.samples})
```

Reply:
Thanks for the tip! Confirmed sorted() will convert the set to a list.